### PR TITLE
Add notes for resolved Cloud bugs

### PR DESCRIPTION
**Pull request by hguthrie**
I added notes for four Resolved Issues.
Requires a technical review.

### CODE REVIEW

File: guides/v2.1/cloud/release-notes/cloud-tools.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ The following updates describe the latest improvements to the `{{site.data.var.c
 
 #### Resolved Issues
 
+-  <!--MAGECLOUD-2736-->Resolved an issue preventing a successful installation of Magento when specifying a custom database configuration.
+
+-  <!--MAGECLOUD-2740-->Fixed an issue that neglected to include each website locale in the `systems` section of the `config.local.php` file after running the `config:dump` command.
+
+-  <!--MAGECLOUD-2797-->Fixed an issue causing warm-up errors during the post-deploy phase by correcting the source base URL reference.
```
Comment (meker12) [inline guides/v2.1/cloud/release-notes/cloud-tools.md]: Consider: that caused warm-up

File: guides/v2.1/cloud/release-notes/cloud-tools.md [hunk 2]
```diff
@@ -40,6 +40,13 @@ The following updates describe the latest improvements to the `{{site.data.var.c
 
 #### Resolved Issues
 
+-  <!--MAGECLOUD-2736-->Resolved an issue preventing a successful installation of Magento when specifying a custom database configuration.
```
Comment (meker12) [inline guides/v2.1/cloud/release-notes/cloud-tools.md]: Consider; an issue that prevented successful...

File: guides/v2.1/cloud/release-notes/cloud-tools.md [hunk 3]
```diff
@@ -40,6 +40,13 @@ The following updates describe the latest improvements to the `{{site.data.var.c
...
+
+-  <!--MAGECLOUD-2740-->Fixed an issue that neglected to include each website locale in the `systems` section of the `config.local.php` file after running the `config:dump` command.
```
Comment (meker12) [inline guides/v2.1/cloud/release-notes/cloud-tools.md]: Consider: Fixed an issue with the `config:dump` CLI command to ensure that it writes website locale data to the `systems` section of the `config.local.php` file.

File: guides/v2.1/cloud/release-notes/cloud-tools.md [hunk 4]
```diff
@@ -40,6 +40,13 @@ The following updates describe the latest improvements to the `{{site.data.var.c
...
+
+-  <!--MAGECLOUD-2797-->Fixed an issue causing warm-up errors during the post-deploy phase by correcting the source base URL reference.
+
+-  <!--MAGECLOUD-2850-->Fixed an issue with the `setup:di:compile` process for Amazon payment module.
```
Comment (meker12) [inline guides/v2.1/cloud/release-notes/cloud-tools.md]: This seems like it needs more info.  I read some of the tickets-- the issue was no read/write permissions on the generated directory after Cloud deployment.

Comment (hguthrie) [reply guides/v2.1/cloud/release-notes/cloud-tools.md]: I have read the tickets and am awaiting development feedback on this.

**Pull request merged**